pkl-cli/pkl-cli.gradle.kts

```diff
@@ -160,47 +160,29 @@ fun Exec.configureExecutable(
         // that the "initialize everything at build time" *CLI* option is likely here to stay
         "--initialize-at-build-time="
         // needed for messagepack-java (see https://github.com/msgpack/msgpack-java/issues/600)
-        ,"--add-opens=java.base/java.nio=ALL-UNNAMED"
-        ,"--add-opens=java.base/sun.nio.ch=ALL-UNNAMED"
+        ,"--initialize-at-run-time=org.msgpack.core.buffer.DirectBufferAccess"
```

Comment:
Have you tested this? I think this would just move our problem from build time to run time.

Reply:
The only way to instantiate a `DirectByteBuffer` is `ByteBuffer.allocateDirect()`, which isn't used in the Pkl codebase or its dependencies. That's why moving this to run time is a cleaner workaround than opening JDK internals.
Is msgpack exercised in native tests? If so, this change should be very safe.

> If DirectByteBuffer is not used, msgpack-java 0.9.3 will work even if these two options are not set.

Reply:
Not right now. Our native tests just use our regular evaluator, and don't exercise the message passing stuff.

Reply:
Looking through msgpack-java, I'm relatively certain that this would end up throwing at runtime: https://github.com/msgpack/msgpack-java/blob/141f66dd615e797c96113169371d1767b251a0ae/msgpack-core/src/main/java/org/msgpack/core/buffer/DirectBufferAccess.java#L286-L289